src/backend/controllers/tankController.js

```diff
@@ -1,68 +1,64 @@
 // @flow strict
 
-// Flow type import
-import type { Request, Response } from 'express';
-
 const User = require('../../models/userModel');
 const Tank = require('../../models/tankModel');
 
 const { validationResult } = require('express-validator');
 
-exports.getFavorite = async (req: Request, res: Response) => {
+// Flow type import
+import type {
+	$Request,
+	$Response,
+	NextFunction,
+	Middleware,
+} from 'express';
+
+exports.getFavorite = async (req: $Request, res: $Response) => {
 	await User.findById(req.user.id, 'favoriteTankId', (err: Error, myUser: User) => {
 		if(err) {
 			res.send(err);
 			console.log('could not find user');
 		} 
-		else {
+		else
```

Comment:
Please keep the curly braces to match our style guide.

(Personally, I don't like the unnecessary ones either, but stuff should be consistent so that it is easier to read)